### DISCUSSION TITLE
Graph background highlighted every 5th and 10th line

### DISCUSSION THREAD

**Discussion by kacago22** [xournalpp/xournalpp#4588] [Ideas]
I'm a math teacher and I use xournal++ for 30 hours a week in class. I often draw lines and / or rectangles while snapping to the grid of the graph background. It would be very useful if there would be a standard graph background where every fifth and tenth grid line would be slightly highlighted. That I could draw longer lines of exact length without having to use the ruler. Thanks for considering this.

**Comment by rolandlo** [comment]
I agree that would be useful.
